### PR TITLE
refactor(exchange.resource): use oui-component

### PR DESCRIPTION
**Pull request by TedKac**

### CODE REVIEW

File: src/exchange/resource/resource.html [hunk 1]
```diff
@@ -69,41 +69,45 @@
...
-                                        data-linkedpopover="exchange/resource/popover/resource.popover.html"
-                                        data-linkedpopover-placement="left"
-                                        data-linkedpopover-remote="true"
-                                        data-linkedpopover-single="true"
-                                        data-linkedpopover-title="{{'common_actions' | translate}}">
-                                    <span class="oui-icon oui-icon-gear_line oui-icon_small" aria-hidden="true"></span>
-                                </button>
+                                <oui-action-menu data-compact 
+                                                 data-align="end"
+                                                 data-disabled="ctrl.services.exchangeStates.constructor.isCreating(resource) || !ctrl.services.exchangeStates.constructor.isOk(resource)">
```
Comment (marie-j) [inline src/exchange/resource/resource.html]: Depending on how both `isCreating` and `isOk` are implemented, maybe you could directly have 
```suggestion
                                                 data-disabled="!ctrl.services.exchangeStates.constructor.isOk(resource)">
```

Comment (marie-j) [inline src/exchange/resource/resource.html]: Depending on how both `isCreating` and `isOk` are implemented, maybe you could directly have 
```suggestion
                                                 data-disabled="!ctrl.services.exchangeStates.constructor.isOk(resource)">
```

Comment (TedKac) [reply src/exchange/resource/resource.html]: Yes, your right, int this case it's possiblre fixed thx 😄

Comment (TedKac) [reply src/exchange/resource/resource.html]: Yes, your right, int this case it's possible fixed thx 😄

Comment (TedKac) [reply src/exchange/resource/resource.html]: Yes, your right, int this case it's possible fixed thx 😄

**Pull request merged**